php/libraries/Utility.class.inc

```diff
@@ -1035,5 +1035,33 @@ class Utility
     {
         return rtrim($path, '/\\') . '/';
     }
+
+    /**
+     * A PHP implementation of python's os.path.join.
```

Comment:
The comment here shouldn't reference other languages, it should just describe what the function does.